dataconnect-sdk/js/default-connector/package.json

```diff
@@ -20,6 +20,6 @@
     "./package.json": "./package.json"
   },
   "peerDependencies": {
-    "firebase": "^11.1.0"
+    "firebase": "^10.14.0 || ^11.0.0"
```

Comment:
if we move to `11.0.0`, we might as well want to update the [root package.json](https://github.com/invertase/tanstack-query-firebase/blob/2ae2aabc9f2bbede3743fe381ca3be765b1dadf2/package.json#L16)